CoreSchemaView/src/au/gov/asd/tac/constellation/views/schemaview/providers/TransactionTypeNodeProvider.java

```diff
@@ -59,11 +63,14 @@
     public static final String MIMETYPE = "application/x-constellation-transactiontype";
     public static final DataFormat TRANSACTION_TYPE = new DataFormat(MIMETYPE);
     private static final int ICON_IMAGE_SIZE = 16;
+    private static final String HIGHLIGHT_BLUE_STYLE = "-fx-background-color: rgb(30,144,255); -fx-padding: 2 5 2 5;";
```

Comment:
@cygnus-x-1 Do you mean a new css file or a java class like JavafxStyleManager? I couldn't find a way how that would support light/dark themes. Is it implemented anywhere in the current code? 
Or can I use Style-Timeline-Light.css and Style-Timeline-Dark.css maybe?

Reply:
I think what @cygnus-x-1 is referring to is if you could try to put this into the `table-view.css` file. There is a separate issue I created to create a light and dark theme and the way we can implement that is to decide which css file we load. That can be done at the generic view framework and what I will do as part of that ticket is to create something like `table-view-light.css` and `table-view-dark.css`. As a prerequisite for that, getting you to put the styles in the css file will help (i.e rather than having it in the code). Noted that there are still other parts of the application that have Java FX styling in code and they need to be fixed too; if we can have new changes not go in the code that will help.

Reply:
Added a similar schema-view.css as these are in Schema View module.